docs/whatsnew-5.1.rst

```diff
@@ -199,12 +248,12 @@ Python 3.5 is not supported for practical reasons.
 Kombu
 -----
 
-Starting from this release, the minimum required version is Kombu 5.0.0.
+Starting from v5.0, the minimum required version is Kombu 5.0.0.
```

Comment:
v5.1 min kombu is also v5.1.0

Reply:
@auvipy Yeah, I left many notes from the last version; since this is a minor version, it's just a reminder that these reqs are still true and started on v5.0.0

Reply:
Ah, wait I think I see what you mean. This should be Kombu 5.1 for Celery 5.1
I'll update this.

Reply:
yeah :D